Test-Source/Invoke-DockerTests.Tests.ps1

```diff
@@ -3,8 +3,20 @@ Import-Module -Force $PSScriptRoot/Docker.Build.Tests.psm1
 Import-Module -Global -Force $PSScriptRoot/MockReg.psm1
 
 . "$PSScriptRoot\..\Source\Private\Utilities.ps1"
+. "$PSScriptRoot\..\Source\Private\Invoke-Command.ps1"
+. "$PSScriptRoot\..\Source\Private\Assert-ExitCodeOk.ps1"
+. "$PSScriptRoot\..\Source\Private\Find-DockerOSType.ps1"
 . "$PSScriptRoot\New-RandomFolder.ps1"
 
+$DockerOsType = Find-DockerOSType
```

Comment:
Consider moving to global scope in the Docker.Tests module, this should be same value across all tests, so one would be able to access it via $Global:DockerOsType